src/utils/windowUtils.ts

```diff
@@ -41,6 +41,35 @@ export module Utils {
       }
     }
 
+    /**
+     * Sends a warning to the console. The warning includes the version number of the deprecation,
+     * the name of the function which was deprecated and an optional message.
+     *
+     * Tu be used in the first line of a deprecated method.
+     *
+     * @param {string} version The version when the tagged method became obsolete
+     * @param {string?} message Optional message to be shown with the warning
+     */
+    export function deprecated(version: string, message = "") {
+
```

Comment:
I personally don't like the newline after the function declaration.  Not a big deal though